blank_project/package.json

```diff
@@ -1,16 +1,22 @@
 {
   "name": "@wasm/hello_world_ts",
   "description": "",
-  "version": "1.0.0",
+  "version": "0.0.1",
   "scripts": {
-    "build": "node_modules/near-shell/near build",
-    "test": "jest test --env=./src/local_test_environment.js",
-    "test-on-devnet": "jest test --env=./src/devnet_test_environment.js"
+    "build": "mkdir -p out/ && gulp",
+    "deploy:contract": "near deploy --account_id=near-hello-devnet --node_url=https://studio.nearprotocol.com/devnet",
```

Comment:
how would this work w/o keys?

Reply:
for now like this:

```
> near-chess@0.0.1 deploy:contract /Users/vg/Documents/near-chess
> near deploy --account_id=near-chess-devnet --node_url=https://studio.nearprotocol.com/devnet

Starting deployment. Account id near-chess-devnet, contract near-chess-devnet, url https://studio.nearprotocol.com/devnet , file ./out/main.wasm
Key lookup failed. Please make sure you set up an account.
```

Reply:
Let's add a helpful error message saying that they should create a different account idm, since they can never actually use near-chess-devnet

Reply:
Generated template would generally have unique contract name, I'm thinking of following workflow.

Create app in `my-awesome-app` folder with `my-awesome-app` contract name:

```
npx create-near-app my-awesome-app
```

`npm install` done behind the scenes (like with `create-react-app`), contract account also created immediately.
This should also eventually have option for different stater templates, etc. 

Run generated app:

```
cd my-awesome-app
npm start
```

Reply:
Converted to separate tracking issue:
https://github.com/nearprotocol/near-shell/issues/25